lib/pilot/build_manager.rb

```diff
@@ -7,9 +7,10 @@ def upload(options)
 
       Helper.log.info "Ready to upload new build to TestFlight (App: #{app.apple_id})...".green
 
-      package_path = PackageBuilder.new.generate(apple_id: app.apple_id,
-                                                 ipa_path: config[:ipa],
-                                             package_path: "/tmp")
+      package_path = FastlaneCore::IpaUploadPackageBuilder.new.generate(app_id: app.apple_id,
+                                                                      ipa_path: config[:ipa],
+                                                                  package_path: "/tmp",
+                                                                      platform: "appletvos")
```

Comment:
I think there's some sort of ipa plist parser available that we might be able to use here? Need to figure out what the platform for the ipa is.